### DISCUSSION TITLE
Getting started instructions error because of Top-Level Await

### DISCUSSION THREAD

**Discussion by jefmsmit** [MichalLytek/type-graphql#1098] [Q&A]
I am using apollo-server-lambda and following the instructions.

I had a working graphql handler before I started introducing type-graphql. 

When following the getting started instructions I get the following error:
`error: Top-level await is currently not supported with the "cjs" output format`

This is from the asynchronous nature of building schema like this:

```
const schema = await buildSchema({
  resolvers: [RecipeResolver],
});
```

Are there other options?

**Comment by MichalLytek** [comment]
Take a look at examples:
https://github.com/MichalLytek/type-graphql/blob/master/examples/simple-usage/index.ts#L8-L28

There's also `buildSchemaSync` if you need.
